packages/commonwealth/client/scripts/views/modals/update_proposal_status_modal.tsx

```diff
@@ -63,6 +75,9 @@ export const UpdateProposalStatusModal = ({
       ]
     : parseCustomStages(customStages);
   const showSnapshot = !!app.chain.meta.snapshot?.length;
+  const isCosmos = app.chain?.chain instanceof CosmosChain;
```

Comment:
We should rely on `app.chain.base === ChainBase.CosmosSDK` here rather than `instanceof`.